### ISSUE TITLE
[Bug] Picker title text is not screen reader accessible

### ISSUE THREAD

**Issue by rachelkang** [dotnet/maui#1186]
### Description

Picker title text is not screen reader accessible on Android, Windows. iOS not yet tested.

### Steps to Reproduce

1. Run MAUI sample on Android and Windows.
2. Navigate to semantics page.
3. Turn on screen reader (TalkBack, Narrator respectively)
4. Go to "select a monkey" picker
5. Notice "select a monkey" text is not read aloud by screen reader (however, when a monkey is selected, the monkey is properly announced)

### Expected Behavior
Picker title text should be screen reader accessible

### Actual Behavior
Picker title text is not screen reader accessible

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

**Comment by rachelkang** [comment]
For Android, this may be related to the EditText hint dilemma - where we have found that in certain instances, while the "hint" (not semantic screen reader hint, but EditText placeholder hint) is visible, it does not actually get read aloud by the screen reader. When the picker is opened, the title _is_ read aloud.

**Comment by XamlTest** [comment]
Verified Repro with Visual Studio Enterprise 2022 main 17.2.0 Preview 2.0 [32227.94.main]. Repro Project is available: [1186.zip](https://github.com/dotnet/maui/files/8177015/1186.zip)

**Comment by XamlTest** [comment]
Verified this issue on Visual Studio Enterprise 17.6.0 Preview 2.0, it does not repro on Windows with below Project:
[1186.zip](https://github.com/dotnet/maui/files/11167937/1186.zip)
